controllers/chores.js

```diff
@@ -1,24 +1,133 @@
 const express = require('express');
 const router = express.Router();
+const db = require('../models');
+const logger = require('../logger');
 
 router.get('/get-chores', (req, res) => {
-    res.send('/get-chores working');
+    db.chore.findAll()
+    .then(data => {
+        res.status(200);
+        return res.json(data);
+    })
+    .catch(err => {
+        logger.error(err);
+        res.status(500);
+        return res.json({ errMessage: 'Server Error' });
+    });
 });
 
 router.get('/get-chore-by-id', (req, res) => {
```

Comment:
I think we should change this route name to 'get-chore' and have the id passed in as a path parameter, similar to how you did for update and delete.

Reply:
done and tested, works